.github/workflows/scripts/scanner-update-offline-bundle.sh

```diff
@@ -7,7 +7,7 @@ ROX_PRODUCT_VERSION="$1"
 PRODUCT_VERSION="${ROX_PRODUCT_VERSION}.0"
 declare -A files_to_download=(
     ["v4/vulns.json.zst"]="${SCANNER_V4_DEFS_BUCKET}/vulnerability-bundles/${PRODUCT_VERSION}/vulns.json.zst"
-    ["v4/mapping.zip"]="https://storage.googleapis.com/definitions.stackrox.io/redhat-repository-mappings/mapping.zip"
+    ["v4/mapping.zip"]="https://definitions.stackrox.io/redhat-repository-mappings/mapping.zip"
```

Comment:
Checking: Do we want to make whatever is in front of `storage.googleapis.com` a dependency of this job?

Reply:
Sorry I don't follow. Which thing is "whatever is in front of ..."?

Reply:
`definitions.stackrox.io` routes to `storage.googleapis.com/defiitions.stackrox.io`, why are we not reaching to `storage.googleapis.com/defiitions.stackrox.io` directly?

Reply:
Just to keep it consistent with the rest of the product